packages/@aws-cdk/pipelines/lib/codepipeline/codepipeline.ts

```diff
@@ -340,6 +347,7 @@ export class CodePipeline extends PipelineBase {
       this._pipeline = new cp.Pipeline(this, 'Pipeline', {
         pipelineName: this.props.pipelineName,
         crossAccountKeys: this.props.crossAccountKeys ?? false,
+        uniqueCrossRegionStackName: this.props.uniqueCrossRegionStackName ?? false,
         // This is necessary to make self-mutation work (deployments are guaranteed
         // to happen only after the builds of the latest pipeline definition).
         restartExecutionOnUpdate: true,
```

Comment:
We're missing a unit test in the pipelines module that this is correctly passed.

Reply:
unit tests added.